### PR TITLE
Aggregation of small fixes from old PR

### PR DESCRIPTION
**Pull request by AndreyTretyak**
Aggregation of small fixes from old PR

### CODE REVIEW

File: src/Extensions/Abstractions/Accessors/Accessor.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace Microsoft.Omex.Extensions.Hosting.Services
+namespace Microsoft.Omex.Extensions.Abstractions.Accessors
```
Comment (artempushkin) [inline src/Extensions/Abstractions/Accessors/Accessor.cs:7]: Maybe Extensions.DependencyInjection.Accessors? How many accessors do we have? Would simply Extensions.DependencyInjection work?

Comment (AndreyTretyak) [reply src/Extensions/Abstractions/Accessors/Accessor.cs:7]: I've put only classes required for Accessor initialization (ex. `Accessor` or `IAccessorSetter`) into this namespace to separate it from `IAccessor` class that needed to consume Accessor.

Comment (AndreyTretyak) [reply src/Extensions/Abstractions/Accessors/Accessor.cs:7]: We could put it into `Microsoft.Omex.Extensions.Abstractions.DependencyInjection ` although I'm not sure what else we would put there

Comment (AndreyTretyak) [reply src/Extensions/Abstractions/Accessors/Accessor.cs:7]: I think it's better to keep it separate

**Pull request merged**